### PR TITLE
[release]Linux/SQL Project

### PR DESCRIPTION
**Pull request by AdityaSharma5**

### CODE REVIEW

File: linux_sql/README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
...
+## Architecture and Design
+![cluster diagram](./assets/cluster_diagram.jpg)
+
+2) Describe tables
+
+host_info: collect the information about the host in the cluster. Each host will be given a serial primary key and the host name will be unique. Other information collected would be cpu number, cpu architecture, cpu model, cpu mhz, L2 cache, total memory and timestamp of when the information is collected. This data will tell the information about the host from which the usage information was also collected.
+
+host_usage: collected the resource information about the each host in the host table. Such information will be free memeory, idle cpu, cpu kernal, disk io, disk available and again a timestamp of when this information is collected. It will also have the host_id from the host_info table to figure out which host this information belongs to.
+
+3) Describe scripts
```
Comment (jarviscanada) [inline linux_sql/README.md]: where is `1)`?? you could use level 3 title instead, e.g. `###`. 

could you please also proofread your readme file? it's not displayed properly
https://github.com/jarviscanada/jarvis_data_eng_Aditya/tree/b619bba491ef7fa4d35dba0baff2a94ebb535d28/linux_sql

File: linux_sql/README.md [hunk 2]
```diff
@@ -1,2 +1,74 @@
...
+
+psql_docker.sh: create/start/stop docker container that will hold the tables
+ddl.sql: create the host info and usage tables
+host_info.sh: collect host info and insert them into the host_info table
+host_usage.sh: collect usage info and insert them into the host_usage table
+queries.sql: run the queries on host_info and host_usage tables. These queries are such as ranking L2 cache in desending order grouped by cpu number and host id. The other query will figure out the average memory use of each host every 5 minutes.
```
Comment (jarviscanada) [inline linux_sql/README.md]: markdown syntax is wrong please proofread

File: linux_sql/README.md [hunk 3]
```diff
@@ -1,2 +1,74 @@
...
+
+--add this to crontab
+
+`* * * * * bash /home/centos/dev/jrvs/bootcamp/linux_sql/host_agent/scripts/host_usage.sh localhost 5432 host_agent postgres password > /tmp/host_usage.log`
+
+
+## Improvements 
+1) write the query to be able to swithc to host_agent in ddl.sql
+2) write the query to be able to get host node failure information in queries.sql
+3) write better sql queries
```
Comment (jarviscanada) [inline linux_sql/README.md]: in general, your readme file is too short. please read the sample readme file (see slack announcements)

File: linux_sql/scripts/host_info.sh [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+psql_host=$1
+psql_port=$2
+db_name=$3
+psql_user=$4
+psql_password=$5
+hostname=$(hostname -f)
+lscpu_out=`lscpu`
+
+
```
Comment (jarviscanada) [inline linux_sql/scripts/host_info.sh]: remove extra newlines in all files

File: linux_sql/README.md [hunk 5]
```diff
@@ -1,2 +1,74 @@
 # Linux Cluster Monitoring Agent
```
Comment (jarviscanada) [inline linux_sql/README.md:1]: instead of opening a new PR, you should have used the previous PR and resolve all comments

**Review by jarviscanada — CHANGES_REQUESTED** [review]

File: linux_sql/README.md (hunk 2 above)
Comment (AdityaSharma5) [reply linux_sql/README.md]: fixed

File: linux_sql/README.md (hunk 1 above)
Comment (AdityaSharma5) [reply linux_sql/README.md]: fixed

File: linux_sql/README.md (hunk 3 above)
Comment (AdityaSharma5) [reply linux_sql/README.md]: fixed

File: linux_sql/scripts/host_info.sh (hunk 4 above)
Comment (AdityaSharma5) [reply linux_sql/scripts/host_info.sh]: fixed

File: linux_sql/README.md (hunk 5 above)
Comment (AdityaSharma5) [reply linux_sql/README.md:1]: ok, i will resolve these comments

**Pull request merged**